lib/vagrant-service-manager/plugin.rb

```diff
@@ -1,3 +1,5 @@
+require_relative 'providers/docker'
```

Comment:
Can we require_relative 'providers/*' to eliminate the need to edit this file when adding new services?